### DISCUSSION TITLE
how to stream to multiple clients and some doubts about video/audio encoding format

### DISCUSSION THREAD

**Discussion by lucksufe** [LizardByte/community#53] [Sunshine] [answered]
I use andorid pad and phone, but they cannot stream same time, one device connect successfully then the other device try to connect will show a error message. I use this v0.14.0 windows release.[sunshine-windows.exe](https://github.com/SunshineStream/Sunshine/releases/download/v0.14.0/sunshine-windows.exe)

Also I see the console show FFMPEG used when streaming, in my opinion FFMPEG use H263 encoding, can I switch to H264 encoding format. And audio has some noise sound，which config may improve the audio quality?

Thanks a lot.

**Comment by ReenigneArcher** [comment, marked as answer]
Increase the number of channels. https://sunshinestream.readthedocs.io/en/latest/about/advanced_usage.html#channels
